process/ReleaseProcess.md

```diff
@@ -10,6 +32,53 @@ The focus of this document refers to generating stable releases of Zowe. For ble
 
+## Zowe Support Policy
+
+### Version timeframe, Current, Active LTS, Maintenance  
+
+New Zowe versions will enter Current Release status for 6 to 9 months to allow consumers of Zowe technology to test, provide feedback, and adjust to any changes.  After the Current Release phase, Zowe will move to Active Long-Term Support (LTS) status and will be deemed ready by the community for general use. Active LTS will have additional releases with both fixes and enhancements. Following a period of Active LTS, the Zowe version will enter Maintenance for fixes only. 
+
+The combination of Active LTS and Maintenance LTS release is designated as "long-term support”, which provides two guarantees:
+- Critical defects will be fixed. The criteria for what constitutes a critical defect is covered in <provide link to description>. 
+- Extenders who achieve Zowe conformance <!--provide link to Conformance Program description--> for the Long-Term Support version will not need to modify their product for it to remain functional when the Zowe community provides distributions within the release or modification level boundary within the same version.  
+
+The length of Active LTS may vary but the total time period of Active LTS + Maintenance LTS will be at least 24 months. Production applications should only use Active LTS or Maintenance LTS releases due to the contract with extender products remaining functional and the community’s commitment to fix critical defects.
+
+### Version, Release, Modification
+
+Zowe distributions are given a version, release, and modification number.  Zowe 1.11.1 is version one, release 11, and modification one.  Zowe distributions are sequential, so 1.11.2, 1.12.0 or 2.0.0 are all later software distributions than 1.11.1.  Distributions that increment the modification number are created when the community recognizes the need to provide a fix mid iteration.  Distributions that increment the release number represent a rollup of all new content since the previous release distribution.  The frequency of releases distributions is determined by the community sprint cycle and is typically every 4 and 6 weeks.  
+
+Distributions that increment the version number represent a change in the support relationship that a customer using Zowe or a vendor extending Zowe has with the underlying Zowe distribution.  Each Zowe version is associated with a conformance program that determines the contract an extender of Zowe has with the base Zowe distribution.  If an extension has achieved the criteria for Zowe version *n* conformance, then that extension will continue to work unmodified with all subsequent release or modification distributions within the same version n. Version n+1 will have new conformance criteria and a vendor product conformance with version *n* may need to be modified to  continue to work with *n+1*. 
```

Comment:
My understanding was extensions conformant with version `n` will function with `n+1`, but they will not be _conformant_ for `n+1`.  I don't think this paragraph says differently, but I had to read it a few times to make sure. Could we clarify this? Perhaps by differentiating conformance vs functional compatibility?